docs/TuxModularity.md

```diff
@@ -0,0 +1,16 @@
+#Tux Modularity
+####[Implementation](#Implementation)
+##Premise
+>Tux is a completely modular framework. This means that you can use as much or as little of Tux as you want in your own development projects. Whereas, most other frameworks have your require the whole frameweork every time you want to use a feature of it, with Tux you only have to require the Tux module you need and not the entire framework.
+
+##Implementation
+Tux's modular nature allows you to only require specifics modules you need. For example, if you wanted to require React and the default Fade animation component in your particular project, you need only:
```

Comment:
Recommend changing second sentence to the following: "For example, if you want to require React and the default Fade animation component in your particular project, you only need to do the following:".